core/com.b2international.snowowl.datastore.server.tests/src/com/b2international/snowowl/datastore/server/internal/branch/BranchTest.java

```diff
@@ -136,6 +136,44 @@ public void testBehindState() throws Exception {
 	public void testDivergedState() throws Exception {
 		assertState(commit(branchA), commit(main), BranchState.DIVERGED);
 	}
+	
+	@Test
+	public void testValidBranchName() {
```

Comment:
The name of the test method does not reflect its purpose. Please add test methods for the following scenarios (if there are no such methods):
Branch.BranchNameValidator.DEFAULT.checkName()
- test success with temporary branch name
- test success with branch name
- test failure with empty branch name
- test failure with branch name with illegal pattern